scripts/notify-subscribers-of-breach.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+const breachName = process.argv[2];
+if (!breachName) {
+  throw("Usage: node notify-subscribers-of-breach.js \"breach-name\"");
+}
+
+const DBUtils = require("../db/utils");
+const EmailUtils = require("../email-utils");
+
+
+EmailUtils.init();
+
+
+async function notifySubscribersOfNewBreach(breachName) {
+  const breach = await DBUtils.getBreachByName(breachName);
+  console.log("Found breach: ", breach);
+  const breachedSubscribers = await DBUtils.getSubscribersForBreach(breach);
+  console.log(`Found ${breachedSubscribers.length} un-notified subscribers in the breach`);
+  for (const subscriber of breachedSubscribers) {
+    console.log(`Sending email to subscriber ID: ${subscriber.id} ...`);
+    const subject = `${subscriber.email} was involved in the ${breach.name} breach.`;
+    try {
+      await EmailUtils.sendEmail(
+        subscriber.email,
+        subject,
+        "breach_notification",
+        { email: subscriber.email, breach: breach }
```

Comment:
Should this pass the entire `breach` object, or just the `breach.name` property?

I think breach_notification.hbs only has the following:

```
Uh-oh, {{ email }} was involved in {{ breach.name }} data breach.
```

Reply:
This is a habit of mine from Django templates ... I find that returning an entire object to templates tends to be more future-proof. So if we need to change the template to access other object properties or methods in the future, we don't have to change the corresponding route.